.dockerignore

```diff
@@ -0,0 +1,17 @@
+
+**/node_modules
+
+**/Dockerfile
+**/dev.Dockerfile
+docker-compose.yml
+prod.docker-compose.yml
+
+**/__pycache__
+**/*.pyc
+**/*.pyo
+**/*.pyd
+**/.Python
+**/env
+
+.gitignore
+.git
```

Comment:
@Gurr1 Is this really necessary since the docker file doesn't copy over these files in `Dockerfile`.

Reply:
The frontend Dockerfile has:
```
COPY ./src ./src
COPY package.json .
COPY ./public ./public
```

The backend has:
```
COPY . . 
```

But this `.dockerignore` is not relative to the frontend and backend folder, so this file doesn't really do anything right?